apps/ewallet/lib/ewallet/validators/transaction_consumption_validator.ex

```diff
@@ -205,8 +209,6 @@ defmodule EWallet.TransactionConsumptionValidator do
     end
   end
 
-  @spec validate_max_consumptions_per_user(%TransactionRequest{}, %Wallet{}) ::
```

Comment:
😢

Reply:
Will put it back, I think I had some issues with Dialyzer and ragedeleted the line.